adapter/config/types.go

```diff
@@ -417,6 +432,15 @@ type APICtlUser struct {
 	Password string
 }
 
+type repository struct {
+	URL string
+	Branch string
+	Username string
+	AccessToken string
+	// SSHKeyFile path to the private key file
+	SSHKeyFile string
```

Comment:
Shall we indent the struct fields ? (Seems due to the comment indention not applied here.) We can place the comment after indenting the fields or else we can place it in front of the field.

Shall we correct in here also ?

- https://github.com/wso2/product-microgateway/blob/57130c2c2fbd2bd48441ad91b8cc46098e58da4b/adapter/internal/loggers/logger.go#L49
- adapter/pkg/loggers/logger.go 

Shall we check indentations in other files also (method documentations, imports etc.) ?

Reply:
Fixed with 763b649